app/code/Bulbulatory/Recommendations/etc/config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <recommendations>
+            <general>
+                <enable>1</enable>
```

Comment:
Jak dodajesz nową funkcjonalność, to najlepiej aby była ona domyślnie wyłączona - klient sam powinien stwierdzić, kiedy najlepiej ją włączyć i samemu wtedy przełączy odpowiedni pstryczek w panelu. Dlatego też dajemy taką możliwość w konfiguracji, aby była nad tym większa kontrola.